lib/html2js.js

```diff
@@ -20,17 +20,39 @@ var escapeContent = function(content) {
   return content.replace(/\\/g, '\\\\').replace(/'/g, '\\\'').replace(/\r?\n/g, '\\n\' +\n    \'');
 };
 
-var createHtml2JsPreprocessor = function(logger, basePath, config) {
+function extend(target) {
+  var sources = [].slice.call(arguments, 1);
+  for (var i=0; i<sources.length; i++) {
+    var src = sources[i];
+    for (var prop in src) {
+      target[prop] = src[prop];
+    }
+  }
+  return target;
+}
```

Comment:
Please use `lodash.extend`, you can pull it directly from https://github.com/karma-runner/karma/blob/master/lib/helper.js